bootstrap.bat

```diff
@@ -1,4 +1,7 @@
-nuget restore
+@echo off
+
+msbuild /t:restore

```

Comment:
Welp. I was playing with .NET Core. I've reverted the change. Would it be prudent to include the nuget 4.1 binary in the root solution folder?